### PR TITLE
nov dev update

### PR DESCRIPTION
**Pull request by korydraughn**
Consider the following:
- TRiRODS
- Globus Connector
- S3 Deep Archive

### CODE REVIEW

**Comment by korydraughn** [comment]
I could add a line or two about TRiRODS and rirods too.

File: content/posts/irods-development-update-november-2022.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Status: published
+
+
+We went to Supercomputing 2022!
+
+We got a chance to talk with Consortium members in person and there was a lot of booth traffic. Overall, it was a success!
+
+Okay, enough about Supercomputing. Let's get into the update.
+
+For the server, we've been working hard to resolve several issues found in the 4.3.0 server. At the top of that list are memory leaks. The server has received some important changes in regard to that. You can follow that work at [https://github.com/irods/irods/pull/6673](https://github.com/irods/irods/pull/6673).
```
Comment (trel) [inline content/posts/irods-development-update-november-2022.md:14]: "memory leaks" -> "memory leaks associated with the new cron manager"

"some important changes and you can follow"

File: content/posts/irods-development-update-november-2022.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+On the plugin side of things, the Audit AMQP plugin (for 4.3.0) has received a huge update. We've switched the plugin from using Qpid Proton's deprecated C API to the supported C++ API. We've also removed the workarounds for logstash and replaced the Jansson JSON library with the Nlohmann JSON library. That is only a subset of the changes introduced during the first pass. Next steps include the handling of multiple AMQP endpoints and re-use of AMQP connections.
+
+Expect a release of the Audit AMQP plugin for iRODS 4.3 soon.
+
+As for clients, we've been hard at work designing and implementing an S3 presentation layer for iRODS. This will allow any S3 client to interact with iRODS. We're making a lot of progress and can't wait to show what we've accomplished. If that sounds interesting, you can stay up to date on this effort by following the repository at [https://github.com/irods/irods_client_s3_cpp](https://github.com/irods/irods_client_s3_cpp).
```
Comment (trel) [inline content/posts/irods-development-update-november-2022.md:22]: "interact with iRODS" -> "interact directly with iRODS"

**Comment by trel** [comment]
let's get s3 deep archive in there, and mention rirods is coming up for trirods

let's leave globus update out for now - still awaiting word on their end.

File: content/posts/irods-development-update-november-2022.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-november-2022.md:14]: The memory leaks are specific to the cron manager. The are across various APIs.

File: content/posts/irods-development-update-november-2022.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-november-2022.md]: done

File: content/posts/irods-development-update-november-2022.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-november-2022.md]: Updated. Let me know if anything needs to be changed. Will mark as resolved if good.

**Comment by korydraughn** [comment]
Added s3 deep archive and TRiRODS.

File: content/posts/irods-development-update-november-2022.md [hunk 3]
```diff
@@ -9,16 +9,20 @@ We went to Supercomputing 2022!
 
 We got a chance to talk with Consortium members in person and there was a lot of booth traffic. Overall, it was a success!
 
-Okay, enough about Supercomputing. Let's get into the update.
+Also, TRiRODS will be happening on Wednesday, December 7 at 2pm ET. Martin Schobben will be talking about the new **rirods** projects he's been working. You can read more about that at [https://irods.org/trirods/](https://irods.org/trirods/).
```
Comment (trel) [inline content/posts/irods-development-update-november-2022.md:12]: "been working" -> "been working on"

"read more about that at" -> "read more at"

File: content/posts/irods-development-update-november-2022.md (hunk 1 above)
Comment (trel) [reply content/posts/irods-development-update-november-2022.md]: resolving.

File: content/posts/irods-development-update-november-2022.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-november-2022.md]: Done

**Comment by korydraughn** [comment]
Will go ahead and squash.

**Comment by korydraughn** [comment]
Squashed.

File: content/posts/irods-development-update-november-2022.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Author: Kory Draughn
+Slug: irods-development-update-november-2022
+Status: published
+
+
+We went to Supercomputing 2022!
+
+We got a chance to talk with Consortium members in person and there was a lot of booth traffic. Overall, it was a success!
+
+Also, TRiRODS will be happening on Wednesday, December 7 at 2pm ET. Martin Schobben will be talking about the new **rirods** projects he's been working on. You can read more at [https://irods.org/trirods/](https://irods.org/trirods/).
```
Comment (korydraughn) [inline content/posts/irods-development-update-november-2022.md:12]: projects -> project

Comment (korydraughn) [reply content/posts/irods-development-update-november-2022.md]: Done

**Comment by trel** [comment]
will merge tomorrow

**Pull request merged**